benchmark/BENCHMARK.md

```diff
@@ -35,9 +35,18 @@ Some scenarios (`CreateDeleteClients`, and `CrawlUsers`, `CreateRealms`, `Create
    * click save
 * Change to the tab `Sevice Account Roles`
    * select for `realm-management` in the `Client Roles` listbox
-   * assign the roles `manage-clients` and `view-users`
+   * assign the roles, based on the below role-mapping table for the respective load simulation scenario
 * the client secret to be passed to the tests can be copied from the `Credentials` tab
 
+| Scenario Name            |       Assigned Roles       |
+|--------------------------|:--------------------------:|
+| CreateDeleteClients      | manage-clients, view-users |
+| CrawlUsers               | manage-clients, view-users |
+| CreateDeleteClientScopes | manage-clients, view-users |
+| CreateDeleteRoles        |       manage-realms        |
```

Comment:
The role `manage-realm` doesn't have an "s" at the end.

```suggestion
| CreateDeleteRoles        |       manage-realm        |
```

Reply:
thanks, good catch

Reply:
done